runtime/lua/vim/lsp/rpc.lua

```diff
@@ -315,8 +315,11 @@ local function start(cmd, cmd_args, dispatchers, extra_spawn_params)
     dispatchers = { dispatchers, 't', true };
   }
 
-  if not (vim.fn.executable(cmd) == 1) then
-    error(string.format("The given command %q is not executable.", cmd))
+  if extra_spawn_params and extra_spawn_params.cwd then
+      assert(is_dir(extra_spawn_params.cwd), "cwd must be a directory")
+  elseif not (vim.fn.executable(cmd) == 1) then
+      error(string.format("The given command %q is not executable.", cmd))
+    end
```

Comment:
I think you forgot to remove the extra `end` after changing the `else if` to `elseif`

Reply:
Should be fixed now :)